source/administration/changelog.md

```diff
@@ -4,6 +4,145 @@ This changelog summarizes updates to [Mattermost Team Edition](http://www.matter
+ - Created an endpoint on iOS to upload a file as octet-stream in order to use `NSUrlSessionUploadTask` that will let us upload files larger than 20 Mb.
+ - Improved caching of `getRootPosts` call.
+ 
+### Channels
+ - Switched search bar to a button and updated it so that clicking it expands the right-hand sidebar with the search box at the top and cursor in place.
+
+### 508 Compliance
+ - Added alt attribute to profile pictures.
+
+### Integrations
+ - Updated incoming webhooks to accept multipart/form-data content type such as that supplied by `curl -F`.
+
+### Notifications
+ - A system message is now posted when a channel is moved between teams by the CLI command.
+
+### System Console
+ - Removed plugin upload setting from System Console UI and prevented switching the setting from the API.
+ - Added paging to system console log viewer and set default value of `per_paging` for logs to 1000.
+ 
+### Enterprise Edition
+ - Added SiteURL for OAuth SSO and fell back to constructed URL if SiteURL blank.
+ 
+## Bug Fixes
+
+ - Fixed an issue where sidebar unreads text setting was ignored in custom theme.
+ - Fixed an issue where emoji picker had an empty line at the bottom of the list.
+ - Fixed an issue with Markdown help wrapping on a second line in Edit Message dialog.
+ - Fixed an issue where after leaving last team the "Logout" link did nothing.
+ - Fixed an issue where focus was sometimes wrong on delete post modal.
+ - Fixed an issue where the bulk import tool didn't force Town Square membership.
+ - Fixed duplicate calls of the "view" request when switching channels.
+ - Fixed an issue where channel name was included in push notifications if someone posted only files with Push Notification Contents set to not include channel name.
+ - Fixed an issue where attempting to preview an attached document failed to finish "loading" if the file extension didn't match the actual file type.
+ - Fixed an issue with no auto-focusing into input text when replying to a message in Classic Mobile App.
+ - Fixed an issue where a username such as "user.name" gets a highlight only on "name" when @-icon is clicked.
+ - Fixed an issue where the "More Unreads Above" indicator didn't always work.
+ - Fixed an issue where IE11 posted placeholder from hidden textbox.
+ - Fixed an issue with not being able to submit a reply with CMD+ENTER "Send messages on CTRL+ENTER" on.
+ - Fixed an issue emojis were difficult to click when trying to add an emoji from the picker.
+ - Fixed an issue where last channel was not remembered after refresh when switching teams.
+ - Fixed an issue with no auto-focusing into input text when attaching a file in Classic Mobile App.
+ - Fixed an issue with not being able to type with composed characters (e.g. CJK) in View Team Members modal and channel switcher.
+ - Fixed an issue where insecure images were loaded by sending client before proxying.
+ - Fixed an issue with sandboxing support for CentOS and Bosh.
```

Comment:
This could be a plugin improvement (Added plugin sandboxing support for CentOS and Bosh)

^Do we have docs for this?

Reply:
I thought they were just bug fixes, but maybe I've forgotten about other PRs than these 2: https://github.com/mattermost/mattermost-server/pull/8418 and https://github.com/mattermost/mattermost-server/pull/8421 - I'll ask Chris on Monday.

Reply:
Chris said they're just bug fixes, not improvements.